man/feh.pre

```diff
@@ -1182,6 +1182,10 @@ file
 .
 .Bl -tag -width indent
 .
+.It %a
+.
+Information about slideshow state (play/pause)
```

Comment:
"playing" and "paused" maybe?

Reply:
Good idea, thanks! Fixed it in d6a1c9ed17e8483f35e2294c39d0950468e567b3.